src/test/java/uk/gov/companieshouse/pscstatement/delta/mapper/StatementMapperTest.java

```diff
@@ -0,0 +1,73 @@
+package uk.gov.companieshouse.pscstatement.delta.mapper;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.io.InputStreamReader;
+import java.lang.reflect.InvocationTargetException;
+import java.time.LocalDate;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+import org.springframework.util.FileCopyUtils;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import uk.gov.companieshouse.api.delta.PscStatement;
+import uk.gov.companieshouse.api.delta.PscStatementDelta;
+import uk.gov.companieshouse.api.psc.Statement;
+import uk.gov.companieshouse.api.psc.StatementLinksType;
+import uk.gov.companieshouse.api.psc.Statement.KindEnum;
+import uk.gov.companieshouse.api.psc.Statement.StatementEnum;
+
+@ExtendWith(SpringExtension.class)
+@ContextConfiguration(classes = { StatementMapperImpl.class})
+public class StatementMapperTest {
+    
+    private ObjectMapper mapper;
+    private PscStatementDelta deltaObject;
+    private PscStatement pscStatement;
+
+    @Autowired
+    StatementMapper statementMapper;
+
+    @BeforeEach
+    public void setUp() throws Exception {
+        mapper = new ObjectMapper();
```

Comment:
Optional change but we could recycle the TestHelper method for building a ChsDelta and pull the PscStatement object out of that since they use the same json file?

Reply:
Required use of object mapper either way, think it's better to leave as is